dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -609,5 +612,62 @@ public void VerifyUIElementsInItemsSource()
                 }
             });
         }
+    
+        [TestMethod]
+        public void VerifyRepeaterDoesNotLeakItemContainers()
```

Comment:
Great idea to check that we actually clear all of the elements! That makes a lot more sense than verifying we only have 10 open at the end of update layout.